src/docs/product/performance/index.mdx

```diff
@@ -45,3 +45,17 @@ Below the graphs are a list of transactions that may be filtered down based on g
 ## Key Transaction View
 
 If you have transactions you frequently return to; you can mark each one as a key transaction in their corresponding [summary views](/product/performance/transaction-summary/#key-transaction). This enables a shortcut to view only Key Transactions by toggling the tab in the upper right corner of the homepage.
+
+## Trends View
+
+Trends surfaces transactions that have had significant changes in their performance over time. This view can be found by toggling the tab in the upper right corner of the homepage.
+
+![trends Homepage](trends-homepage.png)
```

Comment:
The screenshot is missing the key transactions tab

Reply:
:+1: will keep this comment unresolved as a reminder to update this later.

Reply:
So I've updated this to the latest, to save some work I haven't pushed to put the key transaction tab back in since it'll be going away fairly soon